jenkins/plugins.sls

```diff
@@ -6,5 +6,7 @@
     - name: {{plugin}}
 {% if version %}
     - version: '{{version}}'
+    - require:
```

Comment:
I was thinking we might make this a 'watch'? Does jenkins always need a restart after a plugin is installed? Or does it depend which plugin? Also does `service jenkins restart` do a safe restart?